### ISSUE TITLE
Forgot `.add`

### ISSUE THREAD

**Issue by thomcom** [stackgl/gl-mat4#14]
Hiya! I'm noticing that `gl-matrix.mat4` and the docs have the `.add` function, but here it does not. Was it forgotten? Thanks!

**Comment by rreusser** [comment]
Good point. It looks to me like they may have just been missed, so I submitted #15 and added sub() as well. There are no tests in place so that I…… haven't actually run the functions… but I mean… it's hard to mess these up…… right? 😬  If it looks good to you, then I can merge and try to get this published.

**Comment by thomcom** [comment]
Nice, that was fast! Looks good to me.

**Comment by rreusser** [comment]
It's all set and merged to `master`, except I don't have npm credentials on me at the moment, so I'll have to publish later.
